descriptor.md

```diff
@@ -220,3 +229,5 @@ In the following example, the descriptor indicates the type of artifact it is re
 [rfc7230-s2.7]: https://tools.ietf.org/html/rfc7230#section-2.7
 [sha256-vs-sha512]: https://groups.google.com/a/opencontainers.org/forum/#!topic/dev/hsMw7cAwrZE
 [iana]: https://www.iana.org/assignments/media-types/media-types.xhtml
+[blake3]: https://www.ietf.org/archive/id/draft-aumasson-blake3-00.html
```

Comment:
"Expires: 21 January 2025"

That's a little bit worrying :sweat_smile:  any idea where discussion of these things usually happens?  Would it make sense to link to something like https://blake3.io/ instead, or is this the best reference since https://blake3.io/ just redirects to the GitHub repo in the very next URL?

Reply:
https://github.com/BLAKE3-team/BLAKE3-specs

Reply:
https://github.com/openssl/openssl/issues/11613

Reply:
C2SP has a tagged pointer to their copy of the spec, which has me leaning that direction: https://c2sp.org/BLAKE3@v1.0.0

Reply:
Updated.